lab_1_classify_by_unigrams/main.py

```diff
@@ -11,6 +12,14 @@ def tokenize(text: str) -> list[str] | None:
     :param text: a text
     :return: a list of lower-cased tokens without punctuation
     """
+    if not isinstance(text, str):
+        return None
+    text = text.lower()
+    tokens = [token for token in text if token.isalpha()]
```

Comment:
do not create the variable - return as is

Reply:
fixed